pages/platform/databases/connect-kubernetes-to-managed-mysql/guide.en-asia.md

```diff
@@ -0,0 +1,302 @@
+
+In order to customize your install, without having to leave the simplicity of using helm and the Wordpress helm chart, you can simply set some of the [configurable parameters of the WordPress chart](https://github.com/helm/charts/tree/master/stable/wordpress#configuration){.external}. 
+Then you can add it to your `helm install` with the `--set` option (`--set param1=value1,param2=value2`)
+
+Options to set for accessing the Managed MySQL database are
+```
+mariadb.enabled=false
+externalDatabase.host=mysql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net
+externalDatabase.user=avnadmin
+externalDatabase.password=xxxxxxxxx
+externalDatabase.database=defaultdb
+externalDatabase.port=20184
+```
+
+This will install the needed elements (a Wordpress pod for the webserver with the Worpdress PHP code),
+allocate the persistent volumes and initialize the services. And at the end, it will give you the connection parameters for your new Wordpress:
+
+<pre class="console"><code>$ helm install my-wordpress bitnami/wordpress --set allowOverrideNone=true,mariadb.enabled=false,externalDatabase.host=mysql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net,externalDatabase.user=avnadmin,externalDatabase.password=xxxxxxxxx,externalDatabase.database=defaultdb,externalDatabase.port=20184
+NAME: my-wordpress
+LAST DEPLOYED: Thu Dec 23 15:49:33 2021
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+CHART NAME: wordpress
+CHART VERSION: 12.2.7
+APP VERSION: 5.8.2
+
+** Please be patient while the chart is being deployed **
+
+Your WordPress site can be accessed through the following DNS name from within your cluster:
+
+    my-wordpress.default.svc.cluster.local (port 80)
+
+To access your WordPress site from outside the cluster follow the steps below:
+
+1. Get the WordPress URL by running these commands:
+
+  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+        Watch the status with: 'kubectl get svc --namespace default -w my-wordpress'
+
+   export SERVICE_IP=$(kubectl get svc --namespace default my-wordpress --template "{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}")
+   echo "WordPress URL: http://$SERVICE_IP/"
+   echo "WordPress Admin URL: http://$SERVICE_IP/admin"
+
+2. Open a browser and access WordPress using the obtained URL.  
+
+3. Login with the following credentials below to see your blog:
+
+  echo Username: user
+  echo Password: $(kubectl get secret --namespace default my-wordpress -o jsonpath="{.data.wordpress-password}" | base64 --decode)
+
+</code></pre>
+
+> [!Warning]
+> Be sure your MySQL database defaultdb is clean before executing the helm install command. If a previous installation is detected, parameters like user and password won't be updated, so the k8s configuration will mismatch with the database one.
```

Comment:
"> Make sure your MySQL defaultdb database is clean before running the `helm install` command. If a previous installation is detected, settings such as user and password will not be updated, so the configuration inside of the Kubernetes cluster will not match that of the database."